stage1_fly/aci/aci.mk

```diff
@@ -25,18 +26,19 @@ FLY_ACI_DIRS := \
 # main stamp rule - makes sure manifest and deps files are generated
 $(call generate-stamp-rule,$(FLY_ACI_STAMP),$(FLY_ACI_MANIFEST) $(FLY_ACI_MANIFEST_KV_DEPMK_STAMP))
 
-# invalidate generated manifest if version changes
-$(call generate-kv-deps,$(FLY_ACI_MANIFEST_KV_DEPMK_STAMP),$(FLY_ACI_GEN_MANIFEST),$(FLY_ACI_MANIFEST_KV_DEPMK),FLY_ACI_VERSION)
+# invalidate generated manifest if version or arch changes
+$(call generate-kv-deps,$(FLY_ACI_MANIFEST_KV_DEPMK_STAMP),$(FLY_ACI_GEN_MANIFEST),$(FLY_ACI_MANIFEST_KV_DEPMK),FLY_ACI_VERSION,FLY_ACI_ARCH)
```

Comment:
`FLY_ACI_VERSION` and `FLY_ACI_ARCH` should be separated with a space, not comma.